src/controllers/authenticationController.ts

```diff
@@ -113,6 +115,16 @@ export default class AuthenticationController {
         : getProfileResponse;
     }
 
+    const deleteInstallationResponse = await this.notificationService.deleteInstallation(
+      user.fiscal_code
+    );
+    if (deleteInstallationResponse.kind !== "IResponseSuccessJson") {
+      log.error(
```

Comment:
correct me if I'm wrong: if the user haven't given permission to use push notification the call to `createOrUpdateNotification()` never happens and this call here result into an error (but isn't)
If true, just call deleteInstallation without logging anything (or use log.debug())

Reply:
`createOrUpdateNotification()` will be eventually called later (if user provide notification permission). This scenario is about cleaning old installations. No user permissions are required. The error logged here is about an error in deleting installation on Notification Hub (network error, missing installation, etc).

Reply:
yes what I meant (but it's possible I'm wrong) is that if the user has never provided the rights to use push notification the `createOrUpdateNotification()` was never called when reaching this point (on logout) so this call will probably miss the installation (404) on the notification hub

Reply:
Yes, it could be. I'll change `log.error` to `log.debug`

Reply:
Fixed in fd811c7